Cargo.toml

```diff
@@ -1,26 +1,32 @@
 [package]
 name = "deepgram"
-version = "0.0.1"
+version = "0.1.0"
```

Comment:
I wouldn't mind keeping this at 0.0.1 to indicate that it's a very early version. Not that 0.1.0 is not a very early version as well, but I'd like to get a couple of iterations in before choosing 0.1.0. Willing to be overridden on this though @jcdyer

Reply:
The advantage of `0.1.0` is that it allows us to publish a [SemVer compatible update](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html) (`0.1.1`) if/when necessary. I think this is why Cargo uses `0.1.0` as the default first version.

Reply:
I start with 0.1.0, but have never given much thought to why.  I like @yottalogical 's reason here.